vm/src/builtins/pytype.rs

```diff
@@ -320,7 +320,7 @@ impl PyType {
     }
 
     #[pyproperty(magic)]
-    fn qualname(&self, vm: &VirtualMachine) -> PyObjectRef {
+    pub fn qualname(&self, vm: &VirtualMachine) -> PyObjectRef {
```

Comment:
Better to have these `pub(crate)` for now and if required they can be bumped to `pub` later.